### DISCUSSION TITLE
got to stage 5, unable to trigger though

### DISCUSSION THREAD

**Discussion by B41D-E** [skills/.github#553] [Hello GitHub Actions]
as title have made a change to readme file, created a pull request but cannot get the course to trigger and move on.

**Comment by SinanBM** [comment]
Damn. No replies since april? Looks like I'm stuck on that step for eternity 🗡️

**Reply by DKranzMAT** [reply]
Been stuck here for an hour.

Is this purgatory?

**Comment by PyaePhyoKanto** [comment]
I had same problem but figure it out. I typed `-body` instead of `--body`. So, double check your command. 
I checked action log and it show some error and I refer to this doc. https://cli.github.com/manual/gh_pr_comment

Edit: Even though, I solved the issue the action is stucked with "Queued" and it turn out Github is having issues currently. What a coincidence! https://github.com/orgs/community/discussions/143045

**Comment by gospino123-uf** [comment]
TLDR: Click `OK` NOT `Cancel` on commit via the web GUI on GitHub.

I had a similar issue. I believe it is a bug on the commit process when on the web GUI. I would receive the following message after clicking `Commit changes...` then `Commit changes` again: `You have unsaved changes. Do you want to discard them?` I had never seen this prompt before, as I do not use the web GUI often, and I thought I had done something wrong. So, I clicked `Cancel`, and I re-committed. This causes multiple commits, and I believe it interferes with the workflows intended to progress the user.
<img width="442" alt="Commit_Changes_Prompt" src="https://github.com/user-attachments/assets/75c5bce5-5efb-4d1e-bd02-ceb393cc52d1" />

After starting from a fresh repository, I confirmed that clicking `OK` allowed me to progress to completion.
